code/modules/crafting/guncrafting.dm

```diff
@@ -3,13 +3,13 @@
 // PARTS //
 
 /obj/item/weaponcrafting/receiver
-	name = "modular receiver"
+	name = "Modular Receiver"
 	desc = "A prototype modular receiver and trigger assembly for a firearm."
 	icon = 'icons/obj/improvised.dmi'
 	icon_state = "receiver"
 
 /obj/item/weaponcrafting/stock
-	name = "rifle stock"
+	name = "Rifle Stock"
```

Comment:
These two aren't crafting recipe names and should probably stay as they are.

Reply:
That's what I get for editing in public transit, one sec